tests/classes/Utils.js

```diff
@@ -135,7 +135,7 @@ describe('Utils', () => {
 
       expect(() => {
         serverless.utils.readFileSync(tmpFilePath);
-      }).to.throw(new RegExp(`in "${tmpFilePath}"`));
+      }).to.throw(new RegExp('YAMLException:'));
```

Comment:
If you remove the path here that would invalidate the whole test, as we're trying to make sure that the path is actually shown in the error message. Otherwise users might get a yaml parse exception but not see where its happening. Is this necessary on Windows?